src/src/Commands/Time.h

```diff
@@ -3,9 +3,12 @@
 
 class String;
 
+#include <time.h>
```

Comment:
Why is this include needed here? 
Only include in .h files if you need to have some struct as parameter or return value (or member of a class)
Sometimes you can even get away with not including in a .h file by forward declaration of the class you need.
But here you don't need any of those in the .h file, so include it in the .cpp where you actually need it.

Reply:
Removed.

Reply:
Removed.